packages/manager/apps/cloud-connect/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@ovh-ux/manager-cloud-connect-app",
+  "version": "0.0.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/cloud-connect"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "yarn build:eu && yarn build:ca && yarn build:us",
+    "build:ca": "webpack --env.production --env.region='CA' && mv dist dist-CA",
+    "build:eu": "webpack --env.production && mv dist dist-EU",
+    "build:us": "webpack --env.production --env.region='US' && mv dist dist-US",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-cloud-connect-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-cloud-connect-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-cloud-connect-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-cloud-connect": "^0.0.0",
```

```suggestion
    "@ovh-ux/manager-cloud-connect": "^0.0.0 || ^1.0.0",
```
Also we should add a `BREAKING CHANGE` commit on the module initialization to release it as a `1.0.0`